src/components/molecules/VolumePopOverContent/VolumePopOverContent.scss

```diff
@@ -0,0 +1,6 @@
+@import "scss/constants.scss";
+
+// @debt hack-fix to update popover sizes
+#volume-popover {
+  margin-top: $spacing--lg;
+}
```

Comment:
Will some combination of `useEffect` and `useRef` help deal with stubborn popovers?

[OOS]
Some Bootstrap components should probably be reworked as our custom in-house components if that will save us the trouble of constantly searching for hacks and workarounds to make them work